superset/viz.py

```diff
@@ -1852,7 +1852,10 @@ def process_spatial_data_obj(self, key, df):
         elif spatial.get('type') == 'delimited':
             df[key] = (df[spatial.get('lonlatCol')].str.split(spatial.get('delimiter')))
             if spatial.get('reverseCheckbox'):
-                df[key] = [list(reversed(item))for item in df[key]]
+                df[key] = [
+                    tuple(reversed(o)) if isinstance(o, (list, tuple)) else (0, 0)
```

Comment:
Is there a way for us to give the user a warning about that some of the reversed values weren't converted properly?

Reply:
It'd be somewhat tricky from this context. On the JS side we can `notify.warning("Foo bar")` but there are no easy mechanics from this context.